src/lib/expression/between_expression.cpp

```diff
@@ -29,8 +29,7 @@ std::shared_ptr<AbstractExpression> BetweenExpression::deep_copy() const {
 
 std::string BetweenExpression::as_column_name() const {
   std::stringstream stream;
-  stream << _enclose_argument_as_column_name(*value()) << " "
-         << predicate_condition_to_string.left.at(predicate_condition) << " "
+  stream << _enclose_argument_as_column_name(*value()) << " " << predicate_condition << " "
```

Comment:
Can we also `std::cout << *abstract_expression` now?

Reply:
yes, but in this case `_enclose_argument_as_column_name` is necessary to ensure parentheses are printed to make the printed expression non-ambiguous